lbrynet/conf.py

```diff
@@ -213,6 +216,8 @@ def _convert_value(value):
         'USD': {'type': 'fiat'},
     },
     'DB_REVISION_FILE_NAME': 'db_revision',
+    'DEFAULT_PEER_PORT': DEFAULT_PEER_PORT,
+    'DEFAULT_DHT_NODE_PORT': DEFAULT_DHT_NODE_PORT,
```

Comment:
These FIXED_SETTINGS variables seem unused now

Reply:
yeah, you are right. it can be removed